

drivers/net/via-velocity.c:257: warning: `velocity_save_context' declared `static' but never defined
drivers/net/via-velocity.c:258: warning: `velocity_restore_context' declared `static' but never defined

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-power4-akpm/drivers/net/via-velocity.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN drivers/net/via-velocity.c~velocity-warning-fixes-2 drivers/net/via-velocity.c
--- 25-power4/drivers/net/via-velocity.c~velocity-warning-fixes-2	2004-06-19 22:33:09.699829256 -0700
+++ 25-power4-akpm/drivers/net/via-velocity.c	2004-06-19 22:33:55.791822208 -0700
@@ -254,8 +254,6 @@ static void enable_flow_control_ability(
 static void enable_mii_autopoll(struct mac_regs * regs);
 static int velocity_mii_read(struct mac_regs *, u8 byIdx, u16 * pdata);
 static int velocity_mii_write(struct mac_regs *, u8 byMiiAddr, u16 data);
-static void velocity_save_context(struct velocity_info *vptr, struct velocity_context *context);
-static void velocity_restore_context(struct velocity_info *vptr, struct velocity_context *context);
 static u32 mii_check_media_mode(struct mac_regs * regs);
 static u32 check_connection_type(struct mac_regs * regs);
 static void velocity_init_cam_filter(struct velocity_info *vptr);
_
